### PR TITLE
Add seedToPrivateKey() and seedToAccount()

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add seedToPrivateKey() and seedToAccount()

Related issues:
- https://github.com/ainblockchain/ain-util/issues/25

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -416,7 +416,7 @@ export const privateToAddress = function(privateKey: Buffer): string {
 }
 
 /**
- * Returns an Account with the given private key.
+ * Returns an account with the given private key.
  * @param {Buffer} privateKey
  * @return {Account}
  */
```
Comment (jiyoung-an) [inline src/index.ts:423]: how about rename this method: `privateToAccount` -> `privateKeyToAccount`?

**Review by jiyoung-an — APPROVED** [review]
LGTM, Thanks!

File: src/index.ts (hunk 1 above)
Comment (platfowner) [reply src/index.ts:423]: Thanks for the suggestion. For now, let me just follow the convention in the file and just add a todo instead.

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an Now merging..

**Pull request merged**